service/src/main/java/zone/cogni/asquare/service/elasticsearch/info/ElasticsearchMetadata.java

```diff
@@ -122,4 +122,26 @@ public int hashCode() {
     }
   }
 
+  public static class Configuration {
+
+    private int connectTimeout;
+    private int readTimeout;
```

Comment:
add a default config here
- user of the lib can work with default values if there is nothing specific in the users project
- if we add new config values, it won't break 
- we can still use @ConfigurationProperties to initialise only the values we want to overwrite